compiler/include/symbol.h

```diff
@@ -776,4 +776,33 @@ extern Symbol *gSingleVarAuxFields;
 extern std::map<FnSymbol*,int> ftableMap;
 extern std::vector<FnSymbol*> ftableVec;
 
+#define FUNC_NAME_MAX 256
+extern char llvmPrintIrName[FUNC_NAME_MAX+1];
+extern char llvmPrintIrStage[FUNC_NAME_MAX+1];
+
+//Names representations in LLVM IR and C generated code are
+//different from their names in AST. 'llvmPrintIrCName'
+//is place to keep name in LLVM IR and C version of
+//'llvmPrintIrName' variable.
+extern const char *llvmPrintIrCName;
+extern int llvmPrintIrStageNum;
+
+namespace llvmStageNum {
+enum { NOPRINT = 0,
+       NONE,
+       BASIC,
+       FULL,
+       LAST
+     };
+}
+
+extern const char *llvmStageName[llvmStageNum::LAST];
+
+const char *stageNameFromStageNum(int stageNum);
+int stageNumFromStageName(const char* stageName);
```

Comment:
Shouldn't these functions be prefixed with 'llvm' e.g. 'llvmStageNameFromStageNum' ?